user-guide/Advanced_Functionality/DataMiner_Systems/Removing_a_DataMiner_Agent_from_a_DataMiner_System.md

```diff
@@ -19,7 +19,12 @@ Depending on whether the DataMiner Agent is part of a Failover pair or not, a di
 
 1. In the confirmation box, click *Yes*.
 
-1. If you want to keep using the removed DMA as a standalone Agent, restart the DMA.
+1. Stop the Agent you have removed and decommission it.
```

Comment:
Decommissioning is pretty clear to me. This is how chatgpt describes it:

Decommissioning is the process of taking a system, facility, equipment, or infrastructure out of active service, often in a safe, planned, and regulated manner. It typically involves several steps to ensure that the item or facility is no longer operational and, in some cases, repurposed, dismantled, or safely disposed of.

That's at least also how I meant it to be. Once removed from the cluster, the agent should no longer be used and in case of a VM just be removed. In case of bare metal, it must be reset, windows reinstalled maybe even,... Definitely not just be continued as single agent.